### ISSUE TITLE
Experiment with one-cycle learning rate scheduling

### ISSUE THREAD

**Issue by davidrpugh** [kaust-vislab/tensorflow-gpu-data-science-project#13]
See relevant sections of Chapter 11 notebook...

https://github.com/ageron/handson-ml2/blob/master/11_training_deep_neural_networks.ipynb
